### ISSUE TITLE
Health Statistics: include daily screen time?

### ISSUE THREAD

**Issue by orschiro** [slgobinath/SafeEyes#305]
@slgobinath what do you think of including daily screen time to the health statistics plugin and maybe some information when desired daily screen time has been exceeded? 😗

**Comment by slgobinath** [comment]
Ohh man! you read my mind :-)
I am working on it...

**Comment by orschiro** [comment]
Haha, awesome! 👍😄

**Comment by slgobinath** [comment]
Completed with https://github.com/slgobinath/SafeEyes/commit/66a29c602caa64dbaf5a8bac2fd48e8f8ff217a7. Now Safe Eyes shows your screen time. In addition, it shows a green heart (💚) or a broken red heart (💔️) indicating your RSI status.
💚 - You are doing good
💔️ - You either skip a lot of breaks or actively* use the computer for more than 6 hours.
![screenshot from 2019-02-24 11-52-23](https://user-images.githubusercontent.com/4003168/53675719-17cb2c80-3c67-11e9-9cb1-032529cb415b.png)
![screenshot from 2019-02-24 11-51-52](https://user-images.githubusercontent.com/4003168/53675729-329da100-3c67-11e9-9ef7-dd53101ebc8b.png)

**Comment by orschiro** [comment]
Wow, this is great! 🥳

Screen time is reset daily, correct?

-

**Comment by slgobinath** [comment]
With the recent changes, it resets every time you close Safe Eyes. If you have enabled `Persist internal state` option, it will keep all statistics even if you close Safe Eyes.  Those statistics will be reset after a given period (which is 24 hours by default) but you can change the period in Health Stats plugin configuration.

I would appreciate it if you can play with this feature and provide your feedback.

Thanks

**Comment by orschiro** [comment]
Sure, would love to give you some feedback.

Can you provide me with an early pip version that I can use for easy
upgrade/installation? 🤗

-

**Comment by slgobinath** [comment]
Hi,
Please try this command to install the early version:

```
sudo pip3 uninstall safeeyes
sudo pip3 install --index-url=https://test.pypi.org/simple safeeyes
```

**Comment by orschiro** [comment]
Wow, really great!

My only feedback:

Maybe display yesterday's screen time as well?

Otherwise, how would I know my total screen time of yesterday, especially
in comparison to today? Hopefully I am seeing some improvements in terms of
less screen time? 🤔

-

**Comment by slgobinath** [comment]
Hi,
I've tried different ways to show the history (last 5 days) of screen time as a graph. However, so far I couldn't come up with the best solution without breaking the aesthetic look (which I don't want to sacrifice). The way I tried was adding a Unicode graph but it does not vertically align with the line (See the screenshot).
![Screenshot from 2019-03-12 07-10-28](https://user-images.githubusercontent.com/4003168/54195871-32d34300-4496-11e9-8b1d-83789d9cbca0.png)

If you have any ideas, please share them with me.

**Comment by orschiro** [comment]
Hi there 👋🙂

I do like it though!

Maybe even simpler if we do: Screen time: 12:05 (+3:03)

This would mean 3 hours and 3 minutes more than yesterday.

And then maybe skip the graphs for now?

-

**Comment by slgobinath** [comment]
Good idea!

Added with https://github.com/slgobinath/SafeEyes/commit/09ded85ec8859f477725766aa44613a5dc67af0d

**Comment by orschiro** [comment]
You're doing awesome work! 🥳

-

**Comment by slgobinath** [comment]
Hi,

Do you use Xfce? In My Xfce Manjaro, the Unicode doesn't work.
![Screenshot from 2019-03-16 10-39-18](https://user-images.githubusercontent.com/4003168/54476811-d47ccc00-47d7-11e9-86cc-0269d771172d.png)


Also, the CSD title bar does not override the default title bar.
![XFCE](https://user-images.githubusercontent.com/4003168/54476770-89fb4f80-47d7-11e9-997c-e51a5c6be806.png)

**Comment by orschiro** [comment]
@slgobinath I am on Fedora Xfce. Both looking good to me. 

![image](https://user-images.githubusercontent.com/170947/54496043-db483380-48ea-11e9-98f7-d49a92f1d22d.png)

![image](https://user-images.githubusercontent.com/170947/54496048-e4390500-48ea-11e9-8fbb-a0878e245318.png)

**Comment by slgobinath** [comment]
Thank you very much.
I may miss something in my Manjaro environment (it's just a VM for testing purposes). After testing in all my VMs, I will release the next version.

**Comment by orschiro** [comment]
Great! 👍🙂

Glad I could help testing.

-

**Comment by slgobinath** [comment]
Released in 2.0.9

**Comment by AdamPS** [comment]
FYI I had the missing green heart too and on Ubuntu I fixed it by installing package fonts-noto-color-emoji
